simple/util/filehandler.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A generic FileHandler abstraction that allows clients to work seamlessly with 
+local and GCS files and directories.
+"""
+
+import os
+import io
+import logging
+from google.cloud import storage
+
+_GCS_PATH_PREFIX = "gs://"
+
+
+class FileHandler:
```

Comment:
Add comment this is a base class to be extended.

Reply:
Done.